src/app/dice/dice.component.html

```diff
@@ -1,13 +1,13 @@
 <br>
-<mat-slider thumbLabel min="minDice" max="10" [(ngModel)]="numberOfDice"></mat-slider>
+<mat-slider thumbLabel min="minDice" max="10" [(ngModel)]="numberOfDice" ></mat-slider>
 
 <mat-slider thumbLabel min="6" max="25" [(ngModel)]="sides"></mat-slider>
 
-<button mat-fab (click)="throw()">
+<button mat-fab (click)="throw()" color="basic">
     <mat-icon>casino</mat-icon>
 </button>
 
-<ul class="dice-result" [style]="'background-color: ' + colorchanger[nextColor]">
```

Comment:
the idea behind the color changer is that it makes it visible when the user throws the dice but the outcome is the same as the previous roll.